x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_burn_rates.ts

```diff
@@ -29,26 +29,26 @@ export interface UseFetchSloBurnRatesResponse {
 const LONG_REFETCH_INTERVAL = 1000 * 60; // 1 minute
 
 interface UseFetchSloBurnRatesParams {
-  sloId: string;
+  slo: SLOWithSummaryResponse;
   windows: Array<{ name: string; duration: string }>;
   shouldRefetch?: boolean;
 }
 
 export function useFetchSloBurnRates({
-  sloId,
+  slo,
   windows,
   shouldRefetch,
 }: UseFetchSloBurnRatesParams): UseFetchSloBurnRatesResponse {
   const { http } = useKibana().services;
   const { isInitialLoading, isLoading, isError, isSuccess, isRefetching, data, refetch } = useQuery(
     {
-      queryKey: sloKeys.burnRates(sloId),
+      queryKey: sloKeys.burnRates(slo.id, slo.instanceId),
       queryFn: async ({ signal }) => {
         try {
           const response = await http.post<GetSLOBurnRatesResponse>(
-            `/internal/observability/slos/${sloId}/_burn_rates`,
+            `/internal/observability/slos/${slo.id}/_burn_rates`,
             {
-              body: JSON.stringify({ windows }),
+              body: JSON.stringify({ windows, instanceId: slo.instanceId }),
```

Comment:
`slo.instanceId` is potentially undefined. We need to fallback to ALL_VALUE, e.g.
```suggestion
              body: JSON.stringify({ windows, instanceId: slo.instanceId ?? ALL_VALUE }),
```